diamm/serializers/website/source.py

```diff
@@ -68,11 +68,14 @@ def get_related_entity(self, obj):
 
 
 class SourceProvenanceSerializer(ContextDictSerializer):
+    entity = serpy.MethodField()
+    protectorate = serpy.MethodField()
+    region = serpy.MethodField()
     city = serpy.MethodField()
     country = serpy.MethodField()
-    region = serpy.MethodField()
-    protectorate = serpy.MethodField()
-    entity = serpy.MethodField()
+
+    class Meta:
+        ordering = ('entity', 'protectorate', 'region', 'city', 'country')
```

Comment:
Oops. No, it doesn't work. I was looking at the drf docs and must have just been (un)lucky with a couple of tests. Yeah, I'm trying to control the order so I can loop through the results on the source detail page